parser/parser.go

```diff
@@ -64,12 +64,67 @@ type Node interface {
 }
 
 type AST struct {
-	Select  *Select `(   "SELECT"  @@  `
-	Insert  *Insert `  | "INSERT"  @@  `
-	Replace *Insert `  | "REPLACE" @@ )`
-	End     string  `( ";" )?`
+	Select      *Select      `(   "SELECT"         @@`
+	Insert      *Insert      `  | "INSERT"         @@`
+	Replace     *Insert      `  | "REPLACE"        @@`
+	CreateTable *CreateTable `  | "CREATE" "TABLE" @@ ) ";"?`
 }
 
+type CreateTable struct {
+	Table   string              `@(Ident | QuotedIdent) "("`
+	Entries []*CreateTableEntry `@@ ("," @@)* ")"`
+}
+
+func (c *CreateTable) node() {}
+
+type CreateTableEntry struct {
+	GlobalSecondaryIndex  *GlobalSecondaryIndex  `  @@`
+	LocalSecondaryIndex   *LocalSecondaryIndex   `| @@`
+	ProvisionedThroughput *ProvisionedThroughput `| @@`
+	Attr                  *TableAttr             `| @@` // Must be last.
+}
+
+func (c *CreateTableEntry) node() {}
+
+type ProvisionedThroughput struct {
+	ReadCapacityUnits  int64 `"PROVISIONED" "THROUGHPUT" "READ" @Number`
+	WriteCapacityUnits int64 `"WRITE" @Number`
+}
+
+func (p *ProvisionedThroughput) node() {}
+
+type GlobalSecondaryIndex struct {
+	Name                  string                 `"GLOBAL" "SECONDARY" "INDEX" @(Ident | QuotedIdent)`
```

Comment:
Can we make the `SECONDARY` optional, or just remove it? It feels unnecessary. All non-primary indexes are secondary